internal/fsutil/fsutil.go

```diff
@@ -72,3 +72,10 @@ func SetFileModes(root string, dirMode, fileMode int) error {
 		},
 	)
 }
+
+func FileExists(path string) bool {
+	if _, err := os.Stat(path); err != nil {
+		return !errors.Is(err, os.ErrNotExist)
```

Comment:
If you can figure out a better way to test for file existence in Go, then I'm all for it - but I don't know a better solution.   Returning an error from this function makes it unnecessarily complicated in my opinion.

Reply:
I can't figure out a better way, I still think we should not ignore the error.

Reply:
Why? What scenario are you envisioning where `os.ErrNotExist` doesn't mean that the file doesn't exist and how would it negatively effect the overall workflow?  In this case if there is some wider filesystem problem (e.g. a failed mount) I believe the workflow will fail before this point.

Reply:
`os.ErrNotExist` means that the file doesn't exist, but if you need to check it's that type is because there are other types. I'd not ignore those other types, but if you really **really** want to ignore them, I'd do it the other way around, any kind of error means the file doesn't exist.

Reply:
Okay, I've changed `FileExists()` to return false if `os.Stat()` returns any error.